scripts/scaling/predict_olmo_tiny_examples.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import os
+from typing import Dict
+
+import matplotlib.pyplot as plt
+
+from olmo.aliases import PathOrStr
+from olmo.scaling.scaling_laws.curve_fit import (
+    CurveFitConfig,
+    chinchilla_contaminated_fit,
+    chinchilla_fit,
+    get_data,
+    openai_fit,
+    plot_scaling,
+)
+from olmo.util import StrEnum
+
+# from olmo.scaling.scaling_laws.utils import validation
+
+CONFIGS = {
+    "mup-olmo-128-train": {
+        "path": "wandb_outputs/mup-olmo-128-train.csv",
+        "keys": ["train/CrossEntropyLoss"],  # validation
+        "train_step_min": 150,
+        "train_step_max": 550,
+        "eval_step_max": 750,
```

Comment:
These numbers look pretty small. Is that intentional in MUP? My experience is that early loss points can be noisy, possibly due to LR warmup, so I didn't include points prior to 5k or 10k steps. Does MUP have LR warmup?

Reply:
No, this is just an example to test that the code works.